src/main/java/picard/vcf/GenotypeConcordance.java

```diff
@@ -327,11 +327,11 @@ private boolean indexExists(final File vcf) {
                 final String condition = truthVariantContextType + " " + callVariantContextType;
                 final Integer count = unClassifiedStatesMap.getOrDefault(condition, 0) + 1;
                 unClassifiedStatesMap.put(condition, count);
+            } else {
+                // write to the output VCF
+                writer.ifPresent(w -> writeVcfTuple(tuple, w, scheme));
```

Comment:
Note that if we prevent writing unless `stateClassified` is true, we will not write sites with no variation in either original VCF. This means that a site that is HOM REF in both the truth and call VCFs will not be output. (This is currently breaking other GenotypeConcordance tests.) Is the desired behavior to output no variation sites?